target/linux/x86/base-files/lib/preinit/79_move_config

```diff
@@ -2,13 +2,15 @@
 # Copyright (C) 2012-2015 OpenWrt.org
 
 move_config() {
-	local partdev
+	local partdev magic parttype=ext4
 
 	. /lib/upgrade/common.sh
 
 	if export_bootdevice && export_partdevice partdev 1; then
 		mkdir -p /boot
-		mount -t ext4 -o rw,noatime "/dev/$partdev" /boot
+		magic=$(dd if="/dev/$partdev" bs=1 count=3 skip=54 2>/dev/null)
```

Comment:
I might have come up with a function called `getmagic()` which takes a partition name as the only argument, but this is okay...

Reply:
We got those `get_magic_*()` functions in `/lib/upgrade/common.sh`...

Reply:
Why wasn't this addressed? I'm seeing the same magic used at least 3x times already.